.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches-ignore:
+      - "master"
+
+jobs:
+  lint-check:
+    name: Check for ESLint, Formatting, and Types
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cached node_modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Checkout node action
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Install NPM dependencies
+        run: yarn install --frozen-lockfile
+      
+      # - name: Runs compile asset script
+      #   run: ./compile-assets.js # FIXME: Permission denied error
```

Comment:
@ljharb How do I get past the permission error?

Reply:
either `chmod a+x compile-assets.js` and commit the result, or, `node ./compile-assets`